### PR TITLE
add paragraph about removal of conf file option

### PR DESCRIPTION
**Pull request by brwe**

### CODE REVIEW

File: docs/reference/migration/migrate_2_0/removals.asciidoc [hunk 1]
```diff
@@ -99,3 +99,10 @@ The bulk UDP API has been removed.  Instead, use the standard
 
 The merge scheduler is no longer pluggable.
 
+
+==== Custom config file
+
+ The ability to use config files outside the config directory and config files that are not named elasticsearch.yml has been removed. The config file has to be placed in the config
+ directory and must be named elasticsearch.yml.
```
Comment (clintongormley) [inline docs/reference/migration/migrate_2_0/removals.asciidoc]: `elasticsearch.yml`

**Comment by clintongormley** [comment]
How about this:

    ==== Custom config file

    It is no longer possible to specify a custom config file with the `CONF_FILE`
    environment variable, or the `-Des.config`, `-Des.default.config`, or
    `-Delasticsearch.config` parameters.

    Instead, the config file must be named `elasticsearch.yml` and must be located
    in the default `config/` directory, or in the directory specified in the
    `CONF_DIR` environment variable.


And I'd put it in the Settings section here: https://www.elastic.co/guide/en/elasticsearch/reference/2.0/_setting_changes.html

**Comment by brwe** [comment]
CONF_FILE is actually not an env variable - it is only used when elasticsearch is started as service and the variable is set in ` /etc/default/elasticsearch` for deb and `/etc/sysconfig/elasticsearch` for rpm. Or is it then still called environment variable? If not can we do:
```
==== Custom config file

It is no longer possible to specify a custom config file with the `-Des.config`, `-Des.default.config`, or
`-Delasticsearch.config` (starting from command line) or the `CONF_FILE`
setting (for elasticsearch service).

Instead, the config file must be named `elasticsearch.yml` and must be located
in the default `config/` directory, or in the directory specified with `-Des.path.conf` 
(starting from command line) or the `CONF_DIR` setting (for elasticsearch service).
```

**Comment by clintongormley** [comment]
It is still an environment variable.

**Comment by brwe** [comment]
Thanks for the review!

**Pull request merged**